app/src/main/java/com/gathering/android/profile/repo/ProfileRepository.kt

```diff
@@ -9,4 +9,9 @@ interface ProfileRepository {
         photoUri: String?,
         onResponseReady: (ResponseState<UpdateProfileResponse>) -> Unit
     )
+
+    suspend fun updateProfile2(
+        displayName: String?,
+        photoUri: String?
+    )
```

Comment:
This should return `UpdateProfileResponse ` in the response.

Reply:
done